src/App.js

```diff
@@ -1,13 +1,28 @@
 import React from 'react';
+import { Routes, Route, Link } from 'react-router-dom';
 import Calculator from './components/Calculator';
 import Quotes from './components/quotes';
+import Home from './components/Home';
 
 function App() {
   return (
-    <div className="container">
-      <Calculator />
-      <Quotes />
-    </div>
+    <>
+      <header className="header">
+        <h1 className="header-title">Math Magician App</h1>
+        <nav>
+          <ul>
+            <li><Link to="/">Home</Link></li>
+            <li><Link to="/calculator">Calculator</Link></li>
+            <li><Link to="/quotes">Quote</Link></li>
+          </ul>
+        </nav>
+      </header>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/calculator" element={<Calculator />} />
```

Comment:
- [ ] @clin2on3mun great work on your app, however, your calculator page layout doesn't match with the wireframe given, it's missing the `Lets's do some math!` text on the left side, check the comparison below for a better understanding


Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/clin2on3mun/MathMagician_app/assets/61408860/8be46449-aa11-4d01-a02a-fd8a9b015127) | ![image](https://github.com/clin2on3mun/MathMagician_app/assets/61408860/11faf8f1-f30b-4617-8144-3a125a7ac7df)